### ISSUE TITLE
Improve SEO by adding image alt attributes

### ISSUE THREAD

**Issue by MartinA731** [CMU-313/Teedy#21]
The current SEO score of the Teedy is 64. Informative elements should have short, descriptive alt text. This is not the case for one element.

**Comment by MartinA731** [comment]
I would like to work on this please!

**Comment by aruiz2** [comment]
I would like to work on this please!
